app/src/main/java/com/puzzlebench/clean_marvel_kotlin/presentation/MarvelLandingScreen.kt

```diff
@@ -0,0 +1,31 @@
+package com.puzzlebench.clean_marvel_kotlin.presentation
+
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.os.Handler
+import android.view.animation.AlphaAnimation
+import com.puzzlebench.clean_marvel_kotlin.R
+import kotlinx.android.synthetic.main.activity_marvel_landing_screen.logo_marvel
+import kotlinx.android.synthetic.main.activity_marvel_landing_screen.progress_bar_landing_screen
+
+class MarvelLandingScreen : AppCompatActivity() {
```

Comment:
rename the activity as MarvelLandingScreenActivity pls

Reply:
done